### PR TITLE
Add PodWaitForReady for ClickHouse Pod in waitForClickHousePod

### PR DESCRIPTION
**Pull request by yuntanghsu**
Previously, we only check if CurrentRevision is changed to UpdateRevision in waitForClickHousePod when applying a new Flow Visibility manifest. It is possible that the StatefulSet updates the CurrentRevision before the new pod is ready. To make the e2e test robust, we add PodWaitForReady for ClickHouse Pod in waitForClickHousePod.

### CODE REVIEW

File: test/e2e/framework.go [hunk 1]
```diff
@@ -1410,13 +1410,17 @@ func (data *TestData) waitForClickHousePod() error {
 		// The updating is completed when CurrentRevision is updated to UpdateRevision
 		return ss.Status.CurrentRevision == ss.Status.UpdateRevision, nil
 	})
+	clickHousePodName := fmt.Sprintf("%s-0-0-0", clickHousePodNamePrefix)
```
Comment (tushartathgur) [inline test/e2e/framework.go:1413]: Please use the const mentioned as L172. Could you change at L1323 also?

Comment (yuntanghsu) [reply test/e2e/framework.go]: Thanks for reviewing. I've fixed it and also moved clickHouseStatefulSetName to the same place.

**Review by dreamtalen — APPROVED** [review]
LGTM

**Review by tushartathgur — APPROVED** [review]
LGTM

**Pull request merged**